database/skylink.go

```diff
@@ -86,6 +86,8 @@ func (db *DB) FindSkylink(ctx context.Context, skylink skymodules.Skylink) (Skyl
 // MarkPinned marks a skylink as pinned (or no longer unpinned), meaning
 // that Pinner should make sure it's pinned by the minimum number of servers.
 func (db *DB) MarkPinned(ctx context.Context, skylink skymodules.Skylink) error {
+	db.staticLogger.Tracef("Entering MarkPinned. Skylink: '%s'", skylink)
+	defer db.staticLogger.Trace("Exiting MarkPinned.")
```

Comment:
Should these defer statements also have the skylink? So we can confirm which exit statement pertains to which skylink?

Reply:
Can do. It's not strictly necessary because we are only processing one skylink at a time but it doesn't hurt and it also would make it clearer if we start processing more than one skylink in parallel. I'll add it in a F/U/